src/litDataset.ts

```diff
@@ -115,10 +115,72 @@ export async function fetchResourceInfo(
 
   const resourceInfo = parseResourceInfo(response);
 
-  return resourceInfo;
+  return { resourceInfo };
 }
 
-function parseResourceInfo(
+/**
+ * @internal
+ *
+ * This (currently internal) function fetches the ACL indicated in the [[WithResourceInfo]]
+ * attached to a resource.
+ * @param resourceInfo The Resource info with the ACL URL
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch#Parameters).
+ */
+export async function unstable_fetchAcl(
+  resourceInfo: WithResourceInfo,
+  options: Partial<typeof defaultFetchOptions> = defaultFetchOptions
+): Promise<unstable_WithAcl["acl"]> {
+  if (!unstable_hasAccessibleAcl(resourceInfo)) {
+    return {
+      resourceAcl: null,
+      fallbackAcl: null,
+    };
+  }
+  const [resourceAcl, fallbackAcl] = await Promise.all([
+    internal_fetchResourceAcl(resourceInfo, options),
+    internal_fetchFallbackAcl(resourceInfo, options),
+  ]);
+
+  const acl: unstable_WithAcl["acl"] = {
+    fallbackAcl: fallbackAcl,
+    resourceAcl: resourceAcl,
+  };
+
+  return acl;
+}
+
+/**
+ * Experimental: fetch a LitDataset and its associated Access Control List.
+ *
+ * This is an experimental function that fetches both a Resource, the linked ACL Resource (if
+ * available), and the ACL that applies to it if the linked ACL Resource is not available. This can
```

Comment:
Maybe good to make explicit that the fallback ACL can also be null (i.e. that the developer has to deal with that):

```suggestion
 * available), and the ACL that applies to it if the linked ACL Resource is not available (if accessible). This can
```